app/models/fetcher.rb

```diff
@@ -0,0 +1,14 @@
+require 'json'
+require 'net/http'
+
+def get_yql_data(ticker, startDate, endDate)
```

Comment:
Please add comments describing the expected format of the arguments.  E.g.:

# Params:
# ticker a string representing the stock ticker symbol
# startDate and endDate together represent the range of dates for which data should be fetched.
#   startDate is required, endDate may be empty.  If endDate is empty, the current system time will be used.
#   startDate and endDate are both strings in the MM-DD-YYYY format.

Fix up the description of endDate though-- how are "optional" parameters usually handled in ruby?  would the caller leave it out?  pass a null / nil value?  an empty string?